src/core/server/opensearch/opensearch_config.ts

```diff
@@ -146,6 +148,11 @@ const deprecations: ConfigDeprecationProvider = ({ renameFromRoot }) => [
   renameFromRoot('elasticsearch.customHeaders', 'opensearch.customHeaders'),
   renameFromRoot('elasticsearch.shardTimeout', 'opensearch.shardTimeout'),
   renameFromRoot('elasticsearch.requestTimeout', 'opensearch.requestTimeout'),
+  renameFromRoot('elasticsearch.responseMaxHeapPercentage', 'opensearch.responseMaxHeapPercentage'),
```

Comment:
I'm 50/50 on this being added.

Cons: Theses are new configs and never existed so if someone is already modifying their config to include these new fields we should be pushing them to use `opensearch`

Pros: People who haven't migrated this config from `elasticsearch` to `opensearch` still wouldn't have to use this new config.

Reply:
+1 
seems no need to add this line